pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -156,12 +156,18 @@ func (lvm *pmemLvm) DeleteDevice(name string, flush bool) error {
 	if err != nil {
 		return err
 	}
-	if err := ClearDevice(device, flush); err != nil {
+	if err = ClearDevice(device, flush); err != nil {
```

Comment:
Please use `err :=`, it's better Go style.

Reply:
Done.